### ISSUE TITLE
Bad link in docs/core/howto/quotes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4209]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)| @khorn reported|
|-|-|
|Trac ID|trac#4209|
|Type|enhancement|
|Created|2010-01-13 00:44:51Z|

The link to the section on packages in the Python Tutorial no longer works.

Should link to: http://docs.python.org/tutorial/modules.html#packages

Attachments:

* [4209_bad_link_fix.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/bc9/bc9df7db16d7e5bdda4cc0b9d650014304c67dda/03232859092947ebe35a540edefddb8d0b9f4b72.patch) (805 bytes) - added by khorn on 2010-01-13 00:46:10Z - Patch to fix broken link

<details><summary>Searchable metadata</summary>

```
trac-id__4209 4209
type__enhancement enhancement
reporter__khorn khorn
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1263343491000000 1263343491000000
changetime__1263409934000000 1263409934000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_1"></a>|@khorn removed owner|
|-|-|

Patch attached, please review...

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [27978]) Apply 4209_bad_link_fix.patch: Fix the link to the packages section of the Python Tutorial.

Author: khorn
Reviewer: thijs
Fixes: [#4209](https://github.com/twisted/twisted/issues/4209)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
